docs/docs/addons-test-utils.md

```diff
@@ -46,19 +53,60 @@ var ReactTestUtils = require('react-addons-test-utils') // ES5
  - [`scryRenderedComponentsWithType()`](#scryrenderedcomponentswithtype)
  - [`findRenderedComponentWithType()`](#findrenderedcomponentwithtype)
 
-### Shallow Rendering (Experimental)
+* * *
 
-Shallow rendering is an experimental feature that lets you render a component "one level deep" and assert facts about what its render method returns, without worrying about the behavior of child components, which are not instantiated or rendered. This does not require a DOM.
+## Reference
 
-  - [`createRenderer()`](#createrenderer)
-  - [`render()`](#render)
-  - [`getRenderOutput()`](#getrenderoutput)
+### `createRenderer()`
```

Comment:
Done. I moved the step-by-step guide to shallow rendering back to the top, next to the list of shallow rendering methods.

You'll notice the descriptions for the methods in the reference at the bottom are now shorter. My thinking is that these should limit themselves to describing what the method does, and the guide at the top is what brings them together.